msteams-platform/concepts/deploy-and-publish/appsource-publishing-guidance/publish-on-appsource.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Microsoft Teams app approval process guidance
+description: Describes the approval process for getting your app published to the Microsoft Teams app store
+keywords: teams publish store office publishing AppSource
+---
+# AppSource Submission Process
+
+By developing and submitting a Microsoft Teams app, you are subject to the Bot Developer Framework [Terms of Use](https://aka.ms/bf-terms), [Privacy Policy](https://aka.ms/bf-privacy), and [Code of Conduct](https://aka.ms/bf-conduct) for bot, tab, and messaging extension functionality within your app. If your app contains Office 365 Connector functionality, separate terms may also apply as part of your Connector Registration on the [Connectors Developer Dashboard](https://aka.ms/connectorsdashboard).
```

Comment:
This paragraph seems wrong. Probably what was already there, but I don't think it is correct. Check with Priya?